ros/src/foxglove_bridge/package.xml

```diff
@@ -1,36 +1,26 @@
 <?xml version="1.0"?>
 <package format="3">
     <name>foxglove_bridge</name>
-    <version>0.8.5</version>
+    <version>1.0.0</version>
```

Comment:
maybe don't bump the version number just yet - or bump it to 0.9.0. we should tag 1.0.0 once we feel its stable

Reply:
👍